### DISCUSSION TITLE
How is logging out possible?

### DISCUSSION THREAD

**Discussion by trueqbit** [teamhanko/hanko#657] [Q&A] [answered]
I am looking for a way to explicitly log out a user. The documentation for the public API lists the ['logout' endpoint](https://docs.hanko.io/api/public#tag/User-Management/operation/logout) but it returns http status code `404`.

**Comment by FlxMgdnz** [comment, marked as answer]
Hey, we just added the logout function to the main branch and need to make a new release (v0.5.0). The new version will be available very soon on GitHub and Hanko Cloud (probably tomorrow). Sorry for the confusion.

**Reply by trueqbit** [reply]
Hah, a bit too early :)
I realise now that it wasn't available before....
